config_management/roles/dev-tools/tasks/main.yml

```diff
@@ -38,3 +38,6 @@
     dest: /usr/bin
     mode: 0555
     creates: /usr/bin/terraform
+
+- name: copy apt daily override
```

Comment:
Would it be useful for the next maintainer to have a comment similar to this PR's description next to this task?